packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -1042,7 +1042,9 @@ void log(logging.LogRecord event) {
   if (event.level >= logging.Level.SEVERE) {
     globals.printError('${event.loggerName}: ${event.message}$error', stackTrace: event.stackTrace);
   } else if (event.level == logging.Level.WARNING) {
-    globals.printWarning('${event.loggerName}: ${event.message}$error');
+    if (!event.message.contains('No module for')) {
```

Comment:
If the need for this check is temporary, please add a TODO with issue/PR after which this can be removed.

Reply:
Done! This already fixed in dwds so I added a link to the "release new version of dwds" issue.